dynamics-nav/sql-server-query-hints.md

```diff
@@ -11,7 +12,7 @@ author: jswymer
 ---
 # Configuring Query Hints for Optimizing SQL Server Performance with Dynamics NAV
 
-On its own, SQL Server query optimizer will try to select the best execution plan for queries. Most of the time, query optimizer makes the right choice. [Query hints](https://docs.microsoft.com/sql/t-sql/queries/hints-transact-sql-query?view=sql-server-2017) are strategies that can be enforced by the SQL Server query processor to override any execution plan that the query optimizer might select for a query. The [!INCLUDE[server](includes/nav_server_md.md)] instance includes configuration settings that let you enable or disable the use of the following hints on queries in the database:
+On its own, SQL Server query optimizer will try to select the best execution plan for queries. Most of the time, query optimizer makes the right choice. [Query hints](/sql/t-sql/queries/hints-transact-sql-query?view=sql-server-2017&preserve-view=true) are strategies that can be enforced by the SQL Server query processor to override any execution plan that the query optimizer might select for a query. The [!INCLUDE[server](includes/nav_server_md.md)] instance includes configuration settings that let you enable or disable the use of the following hints on queries in the database:
```

Comment:
you can delete '?view=sql-server-2017&preserve-view=true' rather than adding '&preserve-view=true'. So it would be /sql/t-sql/queries/hints-transact-sql-query